pkg/tracing/jaeger/config_yaml.go

```diff
@@ -141,6 +141,7 @@ func getSampler(config Config) tracesdk.Sampler {
 		sampler = jaegerremote.New(config.ServiceName, remoteOptions...)
 	// Fallback always to default (rate limiting).
 	case SamplerTypeRateLimiting:
+		fallthrough
```

Comment:
Should we simply remove the entire `SamplerTypeRateLimiting` case, or do you want to keep it here for documentation reasons?

Reply:
I think it would be nice to keep it, yes.